hudi-integ-test/src/main/java/org/apache/hudi/integ/testsuite/generator/DeltaGenerator.java

```diff
@@ -58,15 +63,15 @@
 
   private static Logger log = LoggerFactory.getLogger(DeltaGenerator.class);
 
-  private DeltaConfig deltaOutputConfig;
+  private DFSDeltaConfig deltaOutputConfig;
```

Comment:
What is the purpose behind this change ?

Reply:
DFSDeltaConfig extends DeltaConfig

The two settings I have added (getInputParallelism and shouldDeleteOldInputData) are in DFSDeltaConfig.